package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "react": "18.3.1",
     "react-native": "0.75.4",
-    "react-native-fairmatic-sdk": "^3.0.0"
+    "react-native-fairmatic-sdk": "^3.0.2"
```

Comment:
Do you want to keep the caret here? if so (which is correct),please add it in the docs too